school_lunch/__manifest__.py

```diff
@@ -28,6 +28,7 @@
         "views/res_partner_views.xml",
         "views/sale_order_views.xml",
         "views/templates.xml",
+        "wizards/guest_reservation.xml",
```

Comment:
As you add a view, it might be interesting to add it in the `cloc_exclude` no ?

Reply:
Sure, but as a new development, shouldn't it be paid for?

Reply:
As discussed, the other views and datas were added to the `cloc_exclude` so, addig the wizard view is ok (and better, make the whole module as free, since it should be free)